tests/system.py

```diff
@@ -46,7 +47,7 @@ def publisher():
     yield pubsub_v1.PublisherClient()
 
 
-@pytest.fixture(scope=u"module")
+@pytest.fixture()
```

Comment:
I'm not sure this bit is correct:  the systests which do *not* expect to close the client will now not use the shared instance.

Reply:
I'm not entirely sure if tests should assume what other tests do (or not do) with the client, they should be independent from each other. To date the client has probably been shared between them to reduce the initialization overhead, and because there have been no state changes.

Since the client state changes are currently limited to the new test only, I'll construct the client(s) there manually. I agree it will probably be more clear.